src/components/ListOfBooks.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Book from './Book';
+
+class ListOfBooks extends React.PureComponent {
+  render() {
+    const books = [
+      {
+        title: 'Book 1',
+        author: 'Author 1',
+      },
+    ];
+
+    return (
+      <div className="book-list-container">
+        <h2>Books</h2>
+        <Book book={books} />
+      </div>
```

Comment:
- [ ] [REQUIRED] Please, try to dynamically create the list of books. Since you got multiple books to render.

## Example

```jsx
<div className="book-list-container">
  <h2>Books</h2>
  {books.map(book => ( 
      <Book book={book} />
  )}
</div>
```

Reply:
You're right! Thanks again, I thought I had to implement it till the next project, but I'll add it right now.